code/drasil-example/pdcontroller/lib/Drasil/PDController/Body.hs

```diff
@@ -118,6 +117,11 @@ si = SI {
   _usedinfodb = usedDB,
    refdb = refDB}
 
+purp :: Sentence
+purp = foldlSent_ [S "provide a model of a", phrase pidC,
+         S "that can be used for the tuning of the gain constants before",
+         S "the deployment of the controller"]
```

Comment:
Can any of these words be used from concepts (e.g., "constants", "controller")?

Reply:
"Constant" and "controller" are not currently defined as concepts in `drasil-example/pdcontroller/lib/Drasil/PDController/Concepts.hs` and are used all over the SRS simply in strings. Should this be an issue?

Reply:
Potentially? I think there are a lot of concepts that we just use as strings, so a more thorough pass of all our examples to "pull out" these concepts could be beneficial, but is also pretty low on the priority because of the effort required for a relatively small amount of benefit. You could make a general issue (tagged as <kbd>[newcomers](https://github.com/JacquesCarette/Drasil/labels/newcomers)</kbd>
 and <kbd>[Low Priority](https://github.com/JacquesCarette/Drasil/labels/Low%20Priority)</kbd>) for doing this so we don't forget, but we would also need to decide on the right time to do this (e.g., when we confident that our current examples won't be extended to include more terms).

Reply:
I see. I opened issue  #3402 for this. I will mark this conversation as resolved now.